exporter/util.go

```diff
@@ -616,6 +616,9 @@ func createListWorkspaceObjectsFunc(objType string, resourceType string, objName
 			if res := ignoreIdeFolderRegex.FindStringSubmatch(object.Path); res != nil {
 				continue
 			}
+			if !ic.MatchesName(object.Path) {
+				continue
+			}
```

Comment:
What's this change?

Reply:
We didn't support `-match` for notebooks/workspace/directories, so I added it... But it will be superseded by #2574